.storybook/preview.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { ChakraProvider } from '@chakra-ui/react';
+import { Provider } from '../src/components';
 
 export const decorators = [
   (Story) => (
-    <ChakraProvider>
+    <Provider network='rinkeby'>
```

Comment:
Any thoughts on making this an env variable so we don't need to change it for a prod push?

Reply:
This won't get pushed to prod. This is just a Storybook config.